src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst

```diff
@@ -1032,26 +1031,40 @@ Price
 *****
 :Apply: pim_catalog_price_collection
 
+Please note that just like number attributes, prices value must be indexed as a string to be captured by the dynamic mapping.
+
 Data model
 ~~~~~~~~~~
-.. code-block:: yaml
+.. code-block:: php
 
-    price-prices:
-        USD-number: 125
-        EUR-number: 110
+    [
+        'values' => [
+            'a_price-prices' => [
+                'ecommerce' => [
+                    'fr_FR' => [
+                        'USD' => '125.0000'
+                        'EUR' => '110.0000'
+                    ]
+                ]
+            ]
+        ]
+    ]
 
 Filtering
 ~~~~~~~~~
 Same operators than ``number`` apply, but by using the full path to the price with its currency.
 
 Example for the ``>`` operator:
-::
 
-.. code-block:: yaml
+.. code-block:: php
 
-range:
-        price-prices.USD-number:
-            gt: 100
+    [
+        'filter' => [
+            'range' => [
+                'price-prices.<all_locales>.<all_channels>.USD' => [ 'gt' => 100 ]
```

Comment:
Not sure about that as Anaël PR is already merged. I think it should be done in this PR.